src/ParallelAlgorithms/ApparentHorizonFinder/HorizonAliases.hpp

```diff
@@ -36,21 +36,22 @@ using vars_to_interpolate_to_target =
                gr::Tags::SpatialRicci<DataVector, Dim, Frame>>;
 
 template <typename Frame>
-using tags_for_observing =
-    tmpl::list<gr::surfaces::Tags::AreaCompute<Frame>,
-               gr::surfaces::Tags::IrreducibleMassCompute<Frame>,
-               ylm::Tags::MaxRicciScalarCompute,
-               ylm::Tags::MinRicciScalarCompute,
-               gr::surfaces::Tags::ChristodoulouMassCompute<Frame>,
-               gr::surfaces::Tags::DimensionlessSpinMagnitudeCompute<Frame>>;
+using tags_for_observing = tmpl::list<
+    gr::surfaces::Tags::AreaCompute<Frame>,
+    gr::surfaces::Tags::IrreducibleMassCompute<Frame>,
+    ylm::Tags::MaxRicciScalarCompute, ylm::Tags::MinRicciScalarCompute,
+    gr::surfaces::Tags::ChristodoulouMassCompute<Frame>,
+    gr::surfaces::Tags::DimensionlessSpinMagnitudeCompute<Frame>,
+    gr::surfaces::Tags::DimensionlessSpinVectorCompute<Frame, Frame>>;
```

Comment:
Damn. This was my bad. I forgot that this tag list is used in the `ObserveTimeSeriesOnSurface` callback, and that requires the types of all tags it observes to be `double`s. I roughly know how to generalize this, but haven't had a chance to implement it. So you can't add the compute tag here, but you can just append it to the list in the pybinding cpp.